packages/nuxt/src/app/nuxt.ts

```diff
@@ -286,6 +286,8 @@ export function createNuxtApp (options: CreateOptions) {
       nuxtApp.callHook('app:chunkError', { error: (event as Event & { payload: Error }).payload })
     })
 
+    window.useNuxtApp = useNuxtApp
```

Comment:
(Maybe we avoid overriding existing util if exists w/without a warning?)

Reply:
The comment above is also little bit confusing https://github.com/nuxt/nuxt/blob/1f8b3fa5e8dc0347cde3b376a600aec2d5ecc985/packages/nuxt/src/app/nuxt.ts#L283

Reply:
```suggestion
    window.useNuxtApp = window.useNuxtApp || useNuxtApp
```